plog-kafka/src/main/java/com/airbnb/plog/kafka/HashPartitioner.java

```diff
@@ -0,0 +1,22 @@
+package com.airbnb.plog.kafka;
+
+import java.util.Arrays;
+import kafka.producer.Partitioner;
+import kafka.utils.VerifiableProperties;
+
+public class HashPartitioner implements Partitioner {
+  public HashPartitioner (VerifiableProperties props) {
+  }
+
+  public int partition(Object key, int numPartitions) {
+    if (key instanceof byte[]) {
```

Comment:
If we assume the key passed is a String, we can then just use the DefaultPartitioner right?

Reply:
Yes